src/core/toType.js

```diff
@@ -0,0 +1,28 @@
+define( function() {
+	"use strict";
+
+	var class2type = {
+		"[object Boolean]": "boolean",
+		"[object Number]": "number",
+		"[object String]": "string",
+		"[object Function]": "function",
+		"[object Array]": "array",
+		"[object Date]": "date",
+		"[object RegExp]": "regexp",
+		"[object Error]": "error",
+		"[object Symbol]": "symbol"
+	};
```

Comment:
Did you make sure listing all those entries directly is smaller than the loop?

Reply:
+10 minified, -3 non-minified

I guess I'll switch it back?

Reply:
Minified gzipped, I assume? Yes, in that case it's better to switch back.

Reply:
Yeah those are both gzip numbers, without gzip the loop one is always smaller.

Reply:
Except... core.js depends on this, so it looks like we can't also depend on core.js...